### PR TITLE
Updated TABL as per name changes

### PR DESCRIPTION
**Pull request by dhairyya**
Changed senders to domains & addresses while spoof has been updated to Spoofed Senders.

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda and @MSFTTracyP could you please approve this for me?

File: microsoft-365/security/office-365-security/tenant-allow-block-list.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ The Tenant Allow/Block List in the Microsoft 365 Defender portal gives you a way
 
 - URLs to block.
 - Files to block.
-- Sender emails or domains to block.
+- Emails domains or addresses to block.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list.md]:
```suggestion
- Email domains or addresses to block.
```

File: microsoft-365/security/office-365-security/tenant-allow-block-list.md [hunk 2]
```diff
@@ -59,13 +59,13 @@ This article describes how to configure entries in the Tenant Allow/Block List i
 
 - The available URL values are described in the [URL syntax for the Tenant Allow/Block List](#url-syntax-for-the-tenant-allowblock-list) section later in this article.
 
-- The Tenant Allow/Block List allows a maximum of 500 entries for senders, 500 entries for URLs, 500 entries for file hashes, and 1024 entries for spoofing (spoofed senders).
+- The Tenant Allow/Block List allows a maximum of 500 entries for domains & addresses, 500 entries for URLs, 500 entries for file hashes, and 1024 entries for spoofing (spoofed senders).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list.md]:
```suggestion
- The Tenant Allow/Block List allows a maximum of 500 entries for domains & addresses, 500 entries for URLs, 500 entries for file hashes, and 1024 entries for spoofing (spoofed senders).
```

File: microsoft-365/security/office-365-security/tenant-allow-block-list.md [hunk 3]
```diff
@@ -59,13 +59,13 @@ This article describes how to configure entries in the Tenant Allow/Block List i
...
 
 - The maximum number of characters for each entry is:
   - File hashes = 64
   - URL = 250
 
-- An entry should be active within 30 minutes.
+- 99.999% entries should be active within 30 minutes. Rest of the scenarios can take upto 24 hours.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list.md]:
```suggestion
- 99.99% of entries should be active within 30 minutes. The rest of the scenarios can take up to 24 hours. 
```

File: microsoft-365/security/office-365-security/tenant-allow-block-list.md [hunk 4]
```diff
@@ -111,14 +111,14 @@ To manage all allows and blocks, see [Add blocks in the Tenant Allow/Block List]
 
 2. Select the tab you want. The columns that are available depend on the tab you selected:
 
-   - **Senders**:
-     - **Value**: The sender domain or email address.
+   - **Domains & addresses**:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list.md:114]:
```suggestion
   - **Domains and addresses**:
```

File: microsoft-365/security/office-365-security/tenant-allow-block-list.md [hunk 5]
```diff
@@ -142,21 +142,21 @@ To manage all allows and blocks, see [Add blocks in the Tenant Allow/Block List]
 
    You can click **Group** to group the results. The values that are available depend on the tab you selected:
 
-   - **Senders**: You can group the results by **Action**.
-   - **Spoofing**: You can group the results by **Action** or **Spoof type**.
+   - **Domains & addresses**: You can group the results by **Action**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list.md:145]:
```suggestion
   - **Domains and addresses**: You can group the results by **Action**.
```

File: microsoft-365/security/office-365-security/tenant-allow-block-list.md [hunk 6]
```diff
@@ -485,6 +485,6 @@ Only messages from that domain *and* sending infrastructure pair are allowed to
 
 ## What to expect after you add an allow or block entry
 
-After you add an allow entry through the Submissions portal or a block entry in the Tenant Allow/Block List, the entry should start working immediately once the entry in active. The entry will mostly be active within 30 minutes, but sometimes it can take upto 24 hours.
+After you add an allow entry through the Submissions portal or a block entry in the Tenant Allow/Block List, the entry should start working immediately once the entry in active. 99.99% of entries will be active within 30 minutes, while rest of the cases it can take upto 24 hours.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-allow-block-list.md]:
```suggestion
After you add an allow entry through the Submissions portal or a block entry in the Tenant Allow/Block List, the entry should start working immediately once the entry is active. 99.99% of entries will be active within 30 minutes, while the rest of the cases can take up to 24 hours.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**